config/base/catalogd/kustomization.yaml

```diff
@@ -1,7 +1,6 @@
 # Does not include the CRD, which must be added separately (it's non-namespaced)
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
-namespace: olmv1-system
```

Comment:
The problem with it is:
If we need to change the name, then we need to change it in all files, right?
Now we change just in one place.